### PR TITLE
dotnetCore_bddfy

### PR DESCRIPTION
**Pull request by mantunesAmido**

### CODE REVIEW

File: Amido.TechTest.NetCore.Bddfy/UserSteps.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+            var location = response.Headers.Location;
+            userId = location.Segments.LastOrDefault();
+        }
+        public void IShouldGetAUserIdForTheNewUser()
+        {
+            Assert.NotNull(userId);
+        }
+
+        public async Task GivenAUserExists()
```
Comment (jpbsheppard) [inline Amido.TechTest.NetCore.Bddfy/UserSteps.cs:66]: Not  a fan of this method, best remove it and then in the test call the create again and then update

**Review by jpbsheppard — CHANGES_REQUESTED** [review]

File: Amido.TechTest.NetCore.Bddfy/UserSteps.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        {
+            var joe = new User()
+            {
+                Name = "Joe",
+                Password = "MyCurrentPassword"
+            };
+            stepsContext.Current.Add("User", joe);
+        }
+
+        public async Task WhenIRequestToCreateAUserAsync()
```
[the remark this answers is not in the document]
Comment (mantunesAmido) [reply Amido.TechTest.NetCore.Bddfy/UserSteps.cs]: Done

File: Amido.TechTest.NetCore.Bddfy/UserSteps.cs [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        }
+
+        public void Dispose()
+        {
+            DisposeAsync();
+        }
+
+
+
+        public void GivenANewUser()
```
[the remark this answers is not in the document]
Comment (mantunesAmido) [reply Amido.TechTest.NetCore.Bddfy/UserSteps.cs]: Done

File: Amido.TechTest.NetCore.Bddfy/UserSteps.cs (hunk 1 above)
Comment (mantunesAmido) [reply Amido.TechTest.NetCore.Bddfy/UserSteps.cs]: Method removed and added the changes requested

**Review by jpbsheppard — APPROVED** [review]

**Pull request merged**